mono_repo.yaml

```diff
@@ -1,8 +1,8 @@
+# Run `mono_repo generate --validate` to check 
+# that everything is up to date
+self_validate: true
+github:
 travis:
```

Comment:
If you want to *migrated* to GitHub actions, delete this line and delete `.travis.yml`

Reply:
sgtm, done.